zbench.zig

```diff
@@ -68,13 +73,27 @@ const Definition = struct {
         if (self.config.hooks.before_each) |hook| hook();
         defer if (self.config.hooks.after_each) |hook| hook();
 
+        var tracking = TrackingAllocator.init(allocator);
```

Comment:
This one i haven't really verified, but maybe avoid creating initialization of `TrackingAllocator` when `self.config.track_allocations` is false.

Reply:
That's what I tried to do initially, but I was struggling to factor out choosing which allocator while not duplicating the rest of the function in an `if`. If the `TrackingAllocator` is defined on the stack inside a sub block it'll be freed outside of that sub block, so I couldn't do something like `const alloc = if (self.config.track_allocations) TrackingAllocator.init(allocator).allocator() else allocator;`. I can try other ways but always creating the tracking allocator was a simple method.

Reply:
Ah okay, understood. No need to change it, but a small comment would be helpful.